corehq/apps/app_manager/templates/app_manager/app_view_release_manager.html

```diff
@@ -31,13 +31,16 @@
   {% initial_page_data 'build_profiles' app.build_profiles %}
   {% initial_page_data 'latest_version_for_build_profiles' latest_version_for_build_profiles %}
   {% initial_page_data 'latestReleasedVersion' latest_released_version %}
+  {% initial_page_data 'master_briefs' master_briefs %}{# linked apps only #}
+  {% initial_page_data 'multiple_masters' multiple_masters %}{# linked apps only #}
```

Comment:
`{# linked apps only #}`
is that needed to add a comment? `<!-- text -->` wont' work?

Reply:
Either works. `<!-- text -->` is a little weird because `initial_page_data` doesn't produce any html right here, so any html comments would just be in a random place on the page by themselves.